src/components/FormElement.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {ForwardedRef, forwardRef, LegacyRef} from 'react';
+import {View, ViewProps} from 'react-native';
+import * as ComponentUtils from '@libs/ComponentUtils';
+
+type FormElementProps = ViewProps;
```

Comment:
```suggestion
type FormElementProps = ViewProps;

```

Reply:
@kubabutkiewicz Hm, do we need this type at all if it's just `ViewProps`?

Reply:
hm , yeah I guess you are right , we dont need that